hilti/src/rt/types/regexp.cc

```diff
@@ -202,8 +203,7 @@ RegExp::RegExp(const std::vector<std::string>& patterns, regexp::Flags flags) :
 }
 
 void RegExp::_newJrx() {
-    if ( _jrx_shared )
```

Comment:
The commit message says "Replace unreachable pattern errors with assertions.", but this just removes the check; same below. That on purpose?

Reply:
Added them back.